### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kroschenko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,9 +64,9 @@ class ScAgentTest(ScAgent):
 
 class ScAgentClassicTest(ScAgentClassic):
     def on_event(self, class_node: ScAddr, edge: ScAddr, action_node: ScAddr) -> ScResult:
-      # ScAgentClassic automatically checks its action
-      ...
-      return ScResult.OK
+        if not self._confirm_action_class(action_node):  # exclusive method for classic agent
+            return ScResult.SKIP
+        ...
```
Comment (FalseR20) [inline README.md:69]: Rollback it. There is no `self._confirm_action_class` now

File: README.md [hunk 2]
```diff
@@ -71,8 +76,6 @@ you should define the identifier of the action class node and arguments of the S
 `event_class` is set to the `question_initiated` keynode by default.
 `event_type` is set to the `ScEventType.ADD_OUTGOING_EDGE` type by default.
 
-**ClassicScAgent checks its action element automatically and doesn't run `on_event` method if checking fails.**
-
```
Comment (FalseR20) [inline README.md:75]: The same

File: README.md [hunk 3]
```diff
@@ -100,8 +103,6 @@ module.add_agent(agent3)
 module.remove_agent(agent3)
 ```
 
-_Note: you don't need remove agents in the end of program._
-
```
Comment (FalseR20) [inline README.md:104]: The same

File: README.md [hunk 4]
```diff
@@ -529,7 +560,7 @@ Utils to work with actions, events and agents
 def check_action_class(action_class: Union[ScAddr, Idtf], action_node: ScAddr) -> bool: ...
 ```
 
-True if action class has connection to action node.
+This function returns **True** if action class has connection to action node.
```
Comment (FalseR20) [inline README.md:563]: Maybe specify that this function should not be used in the ScAgentClassic

File: README.md [hunk 5]
```diff
@@ -696,7 +732,7 @@ assert check_edge(sc_types.EDGE_ACCESS_VAR_POS_PERM, question_finished, action_n
 assert check_edge(sc_types.EDGE_ACCESS_VAR_POS_PERM, question_finished_successfully, action_node)
 ```
 
-# Use-cases
+# Use-cases
```
Comment (FalseR20) [inline README.md:735]: Rollback it

File: README.md [hunk 6]
```diff
@@ -705,3 +741,4 @@ assert check_edge(sc_types.EDGE_ACCESS_VAR_POS_PERM, question_finished_successfu
     - [based on ScAgentClassic](docs/examples/sum_agent_classic.py)
 - Logging examples:
     - [pretty project logging](docs/examples/pretty_logging.py)
+
```
Comment (FalseR20) [inline README.md:744]: Remove empty line

File: README.md [hunk 7]
```diff
@@ -552,12 +588,12 @@ assert check_action_class("some_classification", action_node)
 
 ### Get action arguments
 
+For getting list of action arguments concatenated by `rrel_[1 -> count]` use:
```
Comment (AlexandrZagorskiy) [inline README.md:591]:
```suggestion
For getting list of action arguments concatenated by `rrel_1, ..., rrel_count` use:
```

**Review by FalseR20 — APPROVED** [review]

**Review by AlexandrZagorskiy — APPROVED** [review]

**Pull request merged**